src/monitor/formation_metadata.c

```diff
@@ -773,3 +771,14 @@ AutoFailoverFormationGetDatum(FunctionCallInfo fcinfo, AutoFailoverFormation *fo
 
 	PG_RETURN_DATUM(resultDatum);
 }
+
+/*
+ * ErrorOutForNotFoundPrimary gives an error for when a
+ * primary node is not found in the given formation and group.
+ */
+void ErrorOutForNotFoundPrimary(const char* formationId, int groupId) {
+	ereport(ERROR,
+		(errmsg("Couldn't find the primary node in "
+					"formation \"%s\", group %d",
+							formationId, groupId)));
+}
```

Comment:
I am not fond of this function's name, what about `ReportPrimaryNotFoundError` or something like that?

Then, the formatting of this piece of code isn't per the project standard. Try and run `make indent`, and if that doesn't do the job, see other function definitions around in the file. Mind about the prototype (2 lines), the brackets positions (they must be on their own lines), etc.